sp/models.py

```diff
@@ -182,6 +182,21 @@ def login(self, request, user, saml):
         return import_string(self.login_method)(request, user, self, saml)
 
 
+class IdPUserDefaultValue(models.Model):
+    idp = models.ForeignKey(
+        IdP, verbose_name=_("identity provider"), related_name="user_defaults", on_delete=models.CASCADE
+    )
+    field = models.CharField(max_length=200, unique=True)
```

Comment:
This can't be unique by itself, in case you have two IdPs that each specify defaults. Could be unique_together with the idp FK.

Reply:
Great catch, I pushed up the fix

Reply:
Hi guys, I am building an saml project using saml-sp. I'm stuck where it keeps giving me the error below.
**OneLogin_Saml2_Error at /sso/birthday/slo/
Invalid dict settings: idp_not_found**

Kindly help. Thanks in advance.